data/assets.yaml

```diff
@@ -141,12 +141,15 @@ shapefiles:
 
   - name: ne_10m_admin_0_countries
     url: https://naciscdn.org/naturalearth/5.1.0/10m/cultural/ne_10m_admin_0_countries.zip
+    prj: 3857
```

Comment:
These files aren't web mercator (3857). Instead they need to be re-projected into web mercator from WGS84 lat/lng

Reply:
updated here: 2635448740ded3a712fbb9cc5c61d108ac734ec9